contrib/CODINGSTYLE.md

```diff
@@ -0,0 +1,43 @@
+# Coding Style
+## Introduction
+This is the coding style guide for Modmesh.  
+* [Introduction](#introduction)
+* [Naming](#naming)
+* [Comments](#comments)
+* [Formatting](#formatting)
+* [Class](#class)
+* [Functions](#functions)
+
+### Naming
+1. The class names use Camel case, such as ```class R3DWidget;```. 
+2. The name of the variable / using-declarations uses snake case,  
+such as ```R3DWidget * viewer;``` and ```using size_type = std::size_t;```. 
+3. The QT-related function name uses Camel case,  
+whereas non-QT-related function name uses snake case.
+4. The name of the class member starts with ```m_```.
```

Comment:
```suggestion
4. The name of the class member starts with `m_`.
```

Reply:
What is the difference between ` and ```?

Reply:
` is for inline code, ``` is for code block

Reply:
@ThreeMonth03 check https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax